fhir-server-test/src/test/java/com/ibm/fhir/cli/test/FHIRCliTest.java

```diff
@@ -313,21 +312,21 @@ private void verifyFileContents(String fname, String... expectedMsgs) throws Exc
             }
         }
     }
-    
+
     private void print(String msg) {
         if (debug) {
             System.out.println(msg);
         }
     }
-    
+
     private static String dirPrefix(String fname) {
         return dirPrefix + "/" + fname;
```

Comment:
please switch to File.separator instead of "/"

Reply:
not my codes, but make sense to change.